server/core/config.cc

```diff
@@ -233,6 +234,7 @@ const char *config_listener_params[] =
     CN_SSL_VERSION,
     CN_SSL_CERT_VERIFY_DEPTH,
     CN_SSL_VERIFY_PEER_CERTIFICATE,
+    CN_SESSION_TRACK_TRX_STATE,
```

Comment:
This should be set as a service parameter. Then the DCB and Listener would not have to be modified.

The listeners are a lower level object that are intended to represent a network socket. Services define higher level attributes so the transaction tracking should be put there.